presto-product-tests/bin/run.sh

```diff
@@ -7,6 +7,12 @@ function absolutepath() {
     [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
 
+# If profile is singlenode-ldap, import the root certificate of presto-coordinator
+if [ "$PROFILE" = "singlenode-ldap" ]
```

Comment:
This script must be ready to be runned locally on the host OS. I don't want this script to modify keystores on my host OS. Why can't you just run the tempto in `centos6-java8-oracle-ldap` based container?